webpack.config.js

```diff
@@ -69,13 +70,16 @@ const webpackConfig = {
                 loader: 'svg-inline-loader'
             },
             {
-                test: /\.js$/,
+                test: /\.(?:ts|js)$/,
                 exclude: /\/node_modules\//,
                 use: {
                     loader: 'babel-loader',
                     options: {
                         babelrc: false,
-                        presets: ['@babel/preset-env'],
+                        presets: [
+                            '@babel/preset-env',
```

Comment:
same q as in commercial 

> does the ordering here matter? like should we be running the TS typescript before the env preset?

i'm curious what the babel output looks like with it in this order

Reply:
Verified output is properly transpiled and plays in IE11